tests/conftest.py

```diff
@@ -80,13 +80,19 @@ def user_factory(django_user_model):
     """
 
     def create_user(
-        username=None, is_pi=False, is_superuser=False, first_name=None, last_name=None
+        username=None,
+        is_pi=False,
+        is_superuser=False,
+        first_name=None,
+        last_name=None,
+        email=None,
     ):
         user = django_user_model.objects.create_user(
             username=username or random_string(),
             is_superuser=is_superuser,
             first_name=first_name or random_string(),
             last_name=last_name or random_string(),
+            email=f"{random_string()}@example.com",
```

Comment:
Would be good for this to follow the same pattern as the other arguments to `create_user`. Currently the `email` keyword argument from above is being ignored.